README.md

```diff
@@ -40,4 +40,12 @@ To contribute:
 8. Run this code for upgrade the database after the migrate
     ```bash
     flask db upgrade
-    ```
```

Comment:
can you please add a note to run the whole test using the following command:

`flask test`